frontend/test/metabase/scenarios/models/models-query-editor.cy.spec.js

```diff
@@ -211,7 +211,9 @@ describe("scenarios > models query editor", () => {
 
       cy.findByText(/Syntax error in SQL/).should("be.visible");
 
-      cy.get(".ace_content").type("T 1");
+      cy.get(".ace_content").type(
+        "{backspace}{backspace}{backspace}{backspace}{backspace}",
```

Comment:
Nit:
You can write this as `"{backspace}".repeat("FROM ".length)`

Reply:
Nice, thanks!